getting-started/CONCEPTS/PROMPTS.md

```diff
@@ -102,7 +102,7 @@ This is the user's post: {{ $post }}
 **Javascript**
 
 ```typescript
-app.beforeTurn((context, state) => {
+app.turn("beforeTurn", (context, state) => {
```

Comment:
Thanks for this fix! Right above this line, could you add the following?
// Turn options include "beforeTurn" and "afterTurn"